src/block-kit/MrkDwn.tsx

```diff
@@ -0,0 +1,41 @@
+/** @jsx JSXSlack.h */
+import { JSXSlack } from '../jsx'
+import { ObjectOutput } from '../utils'
+import html from '../html'
+import { BlockComponentProps } from './Blocks'
+
+interface MrkdwnProps extends BlockComponentProps {
+  children: JSXSlack.Children<BlockComponentProps>
+  verbatim: boolean
+}
+
+interface MrkdwnComponentProps {
+  type: 'mrkdwn_component'
+  text: string
+  verbatim: boolean
+}
+
+export const MrkDwn: JSXSlack.FC<MrkdwnProps & {
+  children: JSXSlack.Children<BlockComponentProps>
+}> = ({ children, verbatim }) => {
+  for (const child of JSXSlack.normalizeChildren(children)) {
+    if (typeof child === 'object') {
+      if (child.type === 'img') {
+        throw new Error('<Verbatim> does not support <img> children')
```

Comment:
The detection of `<img>` and `<Image>` children just for `<Context>` block might be redundant because `<Mrkdwn>` can be used anywhere that allowed HTML-like formatting.